cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,23 @@
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
```

Comment:
None of our other CMake files list the condition in `endif()` blocks. Please remove this for consistency.